Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM docker.io/centos:centos7
 
 RUN yum install -y python-requests && \
-    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b stein current-tripleo && \
+    curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - -b train current-tripleo && \
```

Comment:
The commit doesn't match the commit - master isn't a stable branch, so maybe say "Switch to train OpenStack release" or something?

Reply:
odd, I'm pretty sure I amended it
going to push one more time